[freeimage][openexr] Build error on x64-windows-static

Package: freeimage:x64-windows-static@3.18.0#25

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.39.33521.0
-    vcpkg-tool version: 2024-02-07-8a83681f921b10d86ae626fd833c253f4f8c355b
    vcpkg-scripts version: e15d12a77 2024-03-12 (10 hours ago)

**To Reproduce**

`vcpkg install `
**Failure logs**

```
-- Using cached FreeImage3180.zip.
-- Cleaning sources at E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source E:/sandbox/toolchain/vcpkg/downloads/FreeImage3180.zip
-- Applying patch disable-plugins-depending-on-internal-third-party-libraries.patch
-- Applying patch use-external-jpeg.patch
-- Applying patch use-external-jxrlib.patch
-- Applying patch use-external-libtiff.patch
-- Applying patch use-external-openjpeg.patch
-- Applying patch use-external-png-zlib.patch
-- Applying patch use-external-rawlib.patch
-- Applying patch use-external-webp.patch
-- Applying patch use-external-openexr.patch
-- Applying patch use-freeimage-config-include.patch
-- Applying patch fix-function-overload.patch
-- Applying patch use-typedef-as-already-declared.patch
-- Applying patch use-functions-to-override-libtiff-warning-error-handlers.patch
-- Applying patch remove_auto_ptr.patch
-- Applying patch rawlib-build-fix.patch
-- Using source at E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean
-- Configuring x64-windows-static
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: E:/sandbox/toolchain/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe -v
    Working Directory: E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/x64-windows-static-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      E:\sandbox\toolchain\vcpkg\buildtrees\freeimage\config-x64-windows-static-dbg-CMakeCache.txt.log
      E:\sandbox\toolchain\vcpkg\buildtrees\freeimage\config-x64-windows-static-rel-CMakeCache.txt.log
      E:\sandbox\toolchain\vcpkg\buildtrees\freeimage\config-x64-windows-static-out.log

Call Stack (most recent call first):
  E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/freeimage/portfile.cmake:45 (vcpkg_cmake_configure)
  scripts/ports.cmake:175 (include)



```
<details><summary>E:\sandbox\toolchain\vcpkg\buildtrees\freeimage\config-x64-windows-static-out.log</summary>

```
[1/2] "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir ".." "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=E:/sandbox/toolchain/vcpkg/packages/freeimage_x64-windows-static" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_MAKE_PROGRAM=E:/sandbox/toolchain/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=E:/sandbox/toolchain/vcpkg" "-D_VCPKG_INSTALLED_DIR=E:/sandbox/_obj-lib-etc-win/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
FAILED: ../CMakeCache.txt 
"E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir ".." "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=E:/sandbox/toolchain/vcpkg/packages/freeimage_x64-windows-static" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_MAKE_PROGRAM=E:/sandbox/toolchain/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=E:/sandbox/toolchain/vcpkg" "-D_VCPKG_INSTALLED_DIR=E:/sandbox/_obj-lib-etc-win/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF"
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is MSVC 19.39.33521.0
-- The CXX compiler identification is MSVC 19.39.33521.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found ZLIB: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/zlib.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/zlibd.lib (found version "1.3.1")  
-- Found PNG: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/libpng16.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/libpng16d.lib (found version "1.6.43") 
-- Found JPEG: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/jpeg.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/jpeg.lib (found version "62") 
-- Found TIFF: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/tiff.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/tiffd.lib (found version "4.6.0")  
-- Found LibLZMA: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/lzma.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/lzma.lib (found version "5.6.0") 
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE  
-- Found JXR: E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/include/jxrlib  
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- Found LibRaw: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/manual-link/raw.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/manual-link/rawd.lib (found version "0.21.2") 
-- Found Jasper: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/jasper.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/jasperd.lib (found version "4.2.1") 
-- Configuring done (1.8s)
CMake Error at E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/share/openexr/OpenEXRTargets.cmake:101 (set_target_properties):
  The link interface of target "OpenEXR::OpenEXRCore" contains:

    libdeflate::libdeflate_static

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/share/openexr/OpenEXRConfig.cmake:41 (include)
  E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  CMakeLists.txt:20 (find_package)


-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    FETCHCONTENT_FULLY_DISCONNECTED
    _VCPKG_ROOT_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
[2/2] "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-static-dbg" "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=E:/sandbox/toolchain/vcpkg/packages/freeimage_x64-windows-static/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_MAKE_PROGRAM=E:/sandbox/toolchain/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=E:/sandbox/toolchain/vcpkg" "-D_VCPKG_INSTALLED_DIR=E:/sandbox/_obj-lib-etc-win/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DINSTALL_HEADERS=OFF"
FAILED: ../../x64-windows-static-dbg/CMakeCache.txt 
"E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-static-dbg" "E:/sandbox/toolchain/vcpkg/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe" "E:/sandbox/toolchain/vcpkg/buildtrees/freeimage/src/eImage3180-3016f26dee.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=E:/sandbox/toolchain/vcpkg/packages/freeimage_x64-windows-static/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_MAKE_PROGRAM=E:/sandbox/toolchain/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows-static" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=static" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=E:/sandbox/toolchain/vcpkg" "-D_VCPKG_INSTALLED_DIR=E:/sandbox/_obj-lib-etc-win/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DINSTALL_HEADERS=OFF"
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is MSVC 19.39.33521.0
-- The CXX compiler identification is MSVC 19.39.33521.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.39.33519/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found ZLIB: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/zlib.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/zlibd.lib (found version "1.3.1")  
-- Found PNG: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/libpng16.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/libpng16d.lib (found version "1.6.43") 
-- Found JPEG: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/jpeg.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/jpeg.lib (found version "62") 
-- Found TIFF: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/tiff.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/tiffd.lib (found version "4.6.0")  
-- Found LibLZMA: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/lzma.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/lzma.lib (found version "5.6.0") 
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE  
-- Found JXR: E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/include/jxrlib  
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- Found LibRaw: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/manual-link/raw.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/manual-link/rawd.lib (found version "0.21.2") 
-- Found Jasper: optimized;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/lib/jasper.lib;debug;E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/debug/lib/jasperd.lib (found version "4.2.1") 
-- Configuring done (1.8s)
CMake Error at E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/share/openexr/OpenEXRTargets.cmake:101 (set_target_properties):
  The link interface of target "OpenEXR::OpenEXRCore" contains:

    libdeflate::libdeflate_static

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  E:/sandbox/_obj-lib-etc-win/vcpkg_installed/x64-windows-static/share/openexr/OpenEXRConfig.cmake:41 (include)
  E:/sandbox/toolchain/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  CMakeLists.txt:20 (find_package)


-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    FETCHCONTENT_FULLY_DISCONNECTED
    _VCPKG_ROOT_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
ninja: build stopped: subcommand failed.
```
</details>

<details><summary>E:\sandbox\toolchain\vcpkg\buildtrees\freeimage\config-x64-windows-static-rel-CMakeCache.txt.log</summary>

```

--- a/ports/openexr/fix-cmake-package.patch
+++ b/ports/openexr/fix-cmake-package.patch
@@ -1,0 +1,21 @@
+diff --git a/cmake/OpenEXRConfig.cmake.in b/cmake/OpenEXRConfig.cmake.in
+index fbb98c0a..37073a69 100644
+--- a/cmake/OpenEXRConfig.cmake.in
++++ b/cmake/OpenEXRConfig.cmake.in
+@@ -8,11 +8,12 @@ include(CMakeFindDependencyMacro)
+ set(openexr_needthreads @OPENEXR_ENABLE_THREADING@)
+ if (openexr_needthreads)
+   set(THREADS_PREFER_PTHREAD_FLAG ON)
+-  find_dependency(Threads REQUIRED)
++  find_dependency(Threads)
+ endif()
+ unset(openexr_needthreads)
+ 
+-find_dependency(Imath REQUIRED)
++find_dependency(Imath)
++find_dependency(libdeflate)
+ 
+ include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+ check_required_components("@PROJECT_NAME@")
+
+

--- a/ports/openexr/portfile.cmake
+++ b/ports/openexr/portfile.cmake
@@ -4,6 +4,8 @@ vcpkg_from_github(
     REF "v${VERSION}"
     SHA512 6e0a6fdcfae57c6e8b060d9aeed57140d96d39bffe5e40edd6ea5beb06e569323833d07906316ffca05f48e8409d0ea4174e2cd84d554404a4ee432e07d7b5e6
     HEAD_REF main
+    PATCHES
+        fix-cmake-package.patch # https://github.com/AcademySoftwareFoundation/openexr/pull/1674
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS OPTIONS

--- a/ports/openexr/vcpkg.json
+++ b/ports/openexr/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "openexr",
   "version": "3.2.3",
+  "port-version": 1,
   "description": "OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & Magic for use in computer imaging applications",
   "homepage": "https://www.openexr.com/",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6398,7 +6398,7 @@
     },
     "openexr": {
       "baseline": "3.2.3",
-      "port-version": 0
+      "port-version": 1
     },
     "openfbx": {
       "baseline": "2022-07-18",

--- a/versions/o-/openexr.json
+++ b/versions/o-/openexr.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "99e1e8599b1c2601a39f94b511b623ac51878793",
+      "version": "3.2.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "ccd582f25f0177e879cd408487d541865569dd3a",
       "version": "3.2.3",
       "port-version": 0

